### PR TITLE
Add username in btsync, fix wrong host for headless.

### PR DESCRIPTION
**Pull request by ismailsunni**

### CODE REVIEW

File: deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2 [hunk 1]
```diff
@@ -148,7 +148,9 @@ services:
   contexts-btsync:
     environment:
       - SECRET={{ secret_key.contexts_data }}
+      - DEVICE=InaSAFE-Headless-Contexts-{{ remote_user }}
```
Comment (lucernae) [inline deployment/ansible/development/roles/docker_compose/templates/docker-compose.override.yml.j2]: Nice gan!

File: deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2 [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <env name="DATABASE_PASSWORD" value="docker" />
         <env name="DATABASE_HOST" value="db" />
         <env name="INASAFE_REALTIME_BROKER_URL" value="amqp://guest:guest@{{ inasafe_realtime.broker }}:{{ inasafe_realtime.broker_port }}/" />
-        <env name="INASAFE_HEADLESS_BROKER_URL" value="amqp://guest:guest@{{ inasafe_realtime.broker }}:{{ inasafe_headless.broker_port }}/" />
+        <env name="INASAFE_HEADLESS_BROKER_URL" value="amqp://guest:guest@{{ inasafe_headless.broker }}:{{ inasafe_headless.broker_port }}/" />
```
Comment (lucernae) [inline deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2]: Mantap gan

**Comment by Gustry** [comment]
🇬🇧 English please 🇺🇸

;-)

**Comment by lucernae** [comment]
whooops we got busted :D

**Comment by ismailsunni** [comment]
ckckck, I don't understand what did you say @lucernae :D :D

**Pull request merged**